webui/react/src/components/Markdown.stories.tsx

```diff
@@ -0,0 +1,142 @@
+import React, { useState } from 'react';
+
+import Markdown from './Markdown';
+
+export default {
+  component: Markdown,
+  title: 'Markdown',
+};
+
+const markdown = `# h1 Heading 8-)
+## h2 Heading
+### h3 Heading
+#### h4 Heading
+##### h5 Heading
+###### h6 Heading
+
+
+## Horizontal Rules
+
+___
+
+---
+
+***
+
+
+## Emphasis
+
+**This is bold text**
+
+__This is bold text__
+
+*This is italic text*
+
+_This is italic text_
+
+~~Strikethrough~~
+
+
+## Blockquotes
+
+
+> This is a blockquote
```

Comment:
**chore:** add secondary `>>` and tertiary `>>>` block quotes, since they're cool to see!